plutus-tx/testlib/PlutusTx/Test.hs

```diff
@@ -139,6 +140,19 @@ goldenPirReadable name value =
     . maybe "PIR not found in CompiledCode" (prettyReadableSimple . view progTerm)
     $ getPirNoAnn value
 
+-- | Prints uniques. This should be used sparingly: a simple change to a script or a
+-- compiler pass may change all uniques, making it difficult to see the actual
+-- change if all uniques are printed. It is nonetheless useful sometimes.
+goldenPirReadableU ::
+  (PrettyUni uni, Pretty fun, uni `PLC.Everywhere` Flat, Flat fun) =>
+  String ->
+  CompiledCodeIn uni fun a ->
+  TestNested
+goldenPirReadableU name value =
+  nestedGoldenVsDoc name ".pir"
+    . maybe "PIR not found in CompiledCode" (prettyReadable . view progTerm)
+    $ getPirNoAnn value
+
```

Comment:
Doesn't look to me that you have applied `fourmolu` to this new code (and other new code too).

Reply:
There's no way to only fourmolu a single function, is it? I don't want to fourmolu the whole module, since I only added this one function. We can format it in a separate PR.